### PR TITLE
Add information on how to test and release.

### PR DESCRIPTION
**Pull request by PeterGerrard**
Adds information on how to build, test and release the VSTS DLM Automation Build extension.

### CODE REVIEW

File: Release.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Releasing the VSTS Build Plugin
+## First time set up
+1. You need to create a publisher http://aka.ms/vsmarketplace-manage (Feel free to ignore the "Not Verified" warning. You don't need to be verified to publish to Redgate.)
```
Comment (adrianbanks) [inline Release.md]: What credentials do I use to log in?

File: Release.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+2. You will need to be added as a contributor to the `Redgate Software` publisher.
```
Comment (adrianbanks) [inline Release.md]: How do I do this?

File: Release.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+## Releasing
+
+1. Get the version of the VSTS Build plugin .vsix file that you want to release. You can build this locally.
```
Comment (adrianbanks) [inline Release.md]: Is there a way of getting this from TeamCity, or do I *have* to build it locally?

File: Test.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Uploading for testing
+## First time set up
+1. You need to create a publisher http://aka.ms/vsmarketplace-manage (Feel free to ignore the "Not Verified" warning. Verification is not required to upload private extensions for testing.)
+2. You will need access to either TFS 2015 Update 2 or later or a [Visual Studio Team Services account](https://go.microsoft.com/fwlink/?LinkId=307137&clcid=0x409)
```
Comment (adrianbanks) [inline Test.md]: How do I get access if I don't have it?

File: Test.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+## Building
+1. Install latest version of DLM Automation (Location: http://www.red-gate.com/products/dlm/dlm-automation/)
+2. Check publisher, version and public are correct in extension-manifest.json file
```
Comment (adrianbanks) [inline Test.md]: Where does this file live? Is it part of what has just been installed?

File: Test.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+ - The version must be greater than that on [the marketplace](https://marketplace.visualstudio.com/items?itemName=redgatesoftware.redgateSqlCi)
+ - publisher = \<your personnal publisher>
```
Comment (adrianbanks) [inline Test.md]: What's this? Is it my login, or something else?

File: Test.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+ - public = false
+3. Run build.bat
+4. This will create a \<your publisher>.redgatesqlci-\<version>.vsix file inside the Packages folder
+
+## Test on prem TFS
+1. Navigate to the Team Foundation Server Extensions page on your server. (For example, http://someserver:8080/tfs/_gallery/manage)
```
Comment (adrianbanks) [inline Test.md]: Do we have one of these in Redgate? If so, what is its url and what credentials do I need to log in with?

File: Test.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+2. Click Upload new extension and select the file you have just downloaded.
+3. After the extension has successfully uploaded, click Install and select the Team Project Collection to install into.
+
+## Test using VSTS
+1. Goto http://aka.ms/vsmarketplace-manage
+2. Either Update using the right click menu or select Upload new if not already uploaded.
+3. Select the file created earlier
+4. Click Upload
+5. Click Share and select your VSTS account
+6. Login to the VSTS account and you can test using hosted or local agents (more documentation can be found here https://documentation.red-gate.com/display/DLMA1/Use+the+VSTS+Release+extension)
```
Comment (adrianbanks) [inline Test.md]: What credentials do I use?

File: Test.md (hunk 7 above)
Comment (PeterGerrard) [inline Test.md]: I don't think we currently do.

**Comment by tomcsmith1990** [comment]
Release instructions seem correct :+1:

**Pull request merged**